libc/src/fenv/fegetenv.h

```diff
@@ -9,7 +9,7 @@
 #ifndef LLVM_LIBC_SRC_FENV_FEGETENV_H
 #define LLVM_LIBC_SRC_FENV_FEGETENV_H
 
-#include <fenv.h>
+#include "hdr/fenv_macros.h"
```

Comment:
These functions also use some type definitions from the system header `fenv.h`, not the macros, so you cannot replace the `#include <fenv.h>` here yet.  Same with the implementations.